### PR TITLE
CI: test Python versions 3.6-3.9

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes the CI issue and run tests with versions 3.6-3.9.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         -
             uses: actions/setup-python@v1
             with:
-                python-version: '3.x'
+                python-version: '3.8'
```
Comment (inducer) [inline .github/workflows/ci.yaml:20]: This shouldn't be needed.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
...
```
Comment (inducer) [inline .github/workflows/ci.yaml:32]: We should do a matrix job here for 3.6...3.8.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (matthiasdiener) [reply .github/workflows/ci.yaml]: Fixed.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (matthiasdiener) [reply .github/workflows/ci.yaml]: Good idea, added.

**Review by inducer — APPROVED** [review]
Thx!

**Pull request merged**